### PR TITLE
[stable/external-dns] Add support for Infoblox external-dns

### PR DESCRIPTION
**Pull request by rabadin**
Signed-off-by: Raphael Badin <rabadin@cisco.com>

#### What this PR does / why we need it:

Add support for Infoblox.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by paulczar — CHANGES_REQUESTED** [review]
the chart is missing a maintainers in the Chart.yaml.  You can add yourself to get it to pass tests.

**Comment by rabadin** [comment]
@paulczar Okay, I can do that.  I've seen the error from the CI but I figured it would be a bit weird to put my name since I'm only contributing a tiny change... and since there is no maintainer today things must have landed in the past without any maintainer.

**Comment by rabadin** [comment]
@paulczar Please have a look when you get a chance. I've made the requested change.

File: stable/external-dns/Chart.yaml [hunk 1]
```diff
@@ -3,9 +3,12 @@ description:
   Configure external DNS servers (AWS Route53, Google CloudDNS and others)
   for Kubernetes Ingresses and Services
 name: external-dns
-version: 1.0.2
+version: 1.1.2
```
Comment (linki) [inline stable/external-dns/Chart.yaml]: This should be `1.1.0`

Comment (rabadin) [reply stable/external-dns/Chart.yaml]: Good point. Fixed.

**Pull request merged**